faq/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+
+
+class QuestionCategory(models.Model):
+    name = models.CharField(max_length=50)
+
+    def __str__(self):
+        return self.name
+
+
+class Question(models.Model):
+    question = models.CharField(max_length=255)
+    answer = models.TextField(blank=True)
```

Comment:
It would be a great thing to include `null=True`. Otherwise, Django is going to store an empty string `''` in this field in the database instead of `NULL` when no answer has been provided.